src/five/customerize/customerize.txt

```diff
@@ -107,7 +107,7 @@ the source of its template:
   >>> os.path.basename(template.filename)
   'customize.pt'
 
-  >>> print view.templateCodeFromViewName(u'customizezpt.html') #doctest: +ELLIPSIS
```

Comment:
I think `templateCodeFromViewName` should take care of decoding the code in Python 3; Python code is a native string.

Reply:
done